src/assets/styles/layout.css

```diff
@@ -1,15 +1,204 @@
-body {
+:root {
+  --color-white: rgb(255, 255, 255);
+  --color-black: rgb(0, 0, 0);
+}
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
   height: 100%;
   width: 100%;
   overflow-x: hidden;
   margin: 0;
+  padding: 0;
 }
 
-.full-div {
-  background-image: url(../img/slides/slide-1.jpg);
-  background-size: contain;
-  background-repeat: no-repeat;
-  background-position: center;
-  height: 100vh;
+#root {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+}
+
+body::-webkit-scrollbar {
+  width: 1em;
+}
+ 
+body::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+}
+ 
+body::-webkit-scrollbar-thumb {
+  background-color: darkgrey;
+  outline: 1px solid slategrey;
+}
+
+.portfolio-container {
+  height: 200vh;
   width: 100vw;
+  background: url("../img/slides/slide-1.jpg") center center fixed;
+  background-size: cover;
+}
+
+.welcome-screen {
+  background-color: rgba(0, 0, 0, 0.6);
+  height: 100vh;
+  width: 100%;
+  color: rgb(255, 255, 255);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  text-align: center;
+  line-height: 50px;
+}
+
+.welcome-screen button {
+  width: 6rem;
+  margin: 10rem auto;
+  height: 6rem;
+  background-color: rgb(81, 11, 95);
+  color: rgb(255, 255, 255);
+  font-size: 1.2em;
+  border: none;
+  border-radius: 150px;
+  outline: none;
+  cursor: pointer;
+  transition: linear 0.5s;
+  box-shadow: 15px 15px 15px rgba(0, 0, 0, 0.3);
+}
+
+.welcome-screen button:hover {
+  transform: scale(1.1);
+  background-color: rgba(81, 11, 95, 0.5);
+}
+
+.portfolio-navigation {
+  width: 95%;
+  height: 90vh;
+  margin: 2rem auto;
+  background-color: rgba(0, 0, 0, 0.8);
+  border-radius: 0.5rem;
+  box-shadow: 1rem 1rem 1rem rgba(0, 0, 0, 0.2);
+}
+
+.top-nav {
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+  height: 5%;
+  width: 100%;
+  color: rgb(0, 0, 0);
+  background-color: rgb(215,222,246);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.portfolio-navigation ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  position: relative;
+  top: 0.5rem;
+}
+
+.portfolio-navigation li {
+  margin-right: 1rem;
+}
+
+.close-btn {
+  color: rgb(237,93,87);
+}
+
+.minimize-btn {
+  color: rgb(249,192,61);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)